solc/CommandLineInterface.cpp

```diff
@@ -993,6 +995,16 @@ General Information)").c_str(),
 	;
 	desc.add(optimizerOptions);
 
+	po::options_description smtCheckerOptions("Model Checker Options");
+	smtCheckerOptions.add_options()
+		(
+			g_strModelCheckerEngine.c_str(),
+			po::value<string>()->value_name("none, bmc, chc, all"),
```

Comment:
If you define `g_modelCheckerEngineArgs`, this could be `boost::join(g_modelCheckerEngineArgs, ",")`.